R/pkg/R/generics.R

```diff
@@ -395,6 +396,9 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 
 ####################  SparkDataFrame Methods ########################
 
+#' @param x a SparkDataFrame or GroupedData.
```

Comment:
Hmm.. I see why this would be a place for it. I think it would be easier to maintain if the documentation is next to the function body instead of the generics, but I haven't completely figure the best way to do yet.

The approach we have so far is to keep most of the tag/doc on one of the definition - do you think it would work here?